yarn.lock

```diff
@@ -748,12 +748,12 @@ __metadata:
   linkType: hard
 
 "@aws-sdk/types@npm:^3.222.0":
-  version: 3.496.0
-  resolution: "@aws-sdk/types@npm:3.496.0"
+  version: 3.502.0
```

Comment:
is this an upgrade ? didn't see changes in package.json

Reply:
Wouldn't need to update package.json, as it's a minor upgrade, right? 
`^3.222.0 == (>=3.222.0 && < 2.0.0)`